app/helpers/products_helper.rb

```diff
@@ -39,12 +39,10 @@ def public_calendar_link(product)
   end
 
   def product_url_hint(product)
-    product_url_name = product.try(:url_name).presence || "url-name"
-
     public_send(
       "facility_#{product.class.name.underscore}_url",
       current_facility.url_name,
-      product_url_name,
+      "URL_NAME".html_safe,
```

Comment:
Yeah, this is a hint. It had kind of a weird behavior where on the new form, it's "url-name", but the edit form it was the existing url_name. And then it would update if you changed it after submit, but not before. I wanted to use `<url-name>` or something like that, but Rails was URL-espaping the brackets, e.g. `%3Curl_name%3E`. That's why I went with all caps.